inference-engine/src/mkldnn_plugin/nodes/mkldnn_interpolate_node.h

```diff
@@ -0,0 +1,187 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ie_common.h>
+#include <mkldnn_node.h>
+#include <string>
+#include <memory>
+#include <vector>
+
+#include "ref_eltwise.hpp"
+#include "ref_depthwise.hpp"
+
+using namespace InferenceEngine;
+
+namespace MKLDNNPlugin {
+namespace interpolate {
```

Comment:
Maybe better `Interpolate`?

Reply:
I also don't think we need to put `MKLDNNInterpolateNode` into this namespace.

Reply:
> I also don't think we need to put `MKLDNNInterpolateNode` into this namespace.

do you mean use a more specific name? If so, I changed to name space to `InterpolateNode`. `MKLDNNInterpolateNode` is used with class name, and encapsulated in this name space.

Reply:
I mean we don't put MKLDNN*Node into any namespace expect MKLDNNPlugin. Only enams ans structs needs to be encapsulate into Interpolate namespace.
Another way: don't introduce new namespace but rename `LayoutType`, `CoordTransMode`, `NearestMode`  to `InterpolateLayoutType`, `InterpolateCoordTransMode`, `InterpolateNearestMode` respectively. 
My intention is to make everything we have in header file "Interpolate related" in order to avoid possible conflicts by name in other files where the header will be included.

Reply:
I did with second way, explicitly add prefix. Thanks!